source/declarations.tex

```diff
@@ -2935,8 +2935,12 @@
   using C::x;
   int x(double);
 };
-int f(D* d) {
-  return d->x();    // ambiguous: \tcode{B::x} or \tcode{C::x}
+void f(D* d) {
+   d->x();       // ambiguous: \tcode{x}'s direct parent \tcode{A} is an ambiguous base of the naming class \tcode{D}
```

Comment:
Fair enough - I'm not entirely unsympathetic to that POV.  I might get motivated to re-style it at a future date - but if anyone sees any value in the general idea, feel free to take a crack at it.  Thanks for the feedback.